recipes/libsystemd/all/conanfile.py

```diff
@@ -152,4 +152,6 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["systemd"]
+        # systemd uses only major version in its .pc file
+        self.cpp_info.version = self.version.split('.')[0]
```

Comment:
```suggestion
        # FIXME: this `.version` should only happen for the `pkg_config` generator (see https://github.com/conan-io/conan/issues/8202)
        # systemd uses only major version in its .pc file
        self.cpp_info.version = self.version.split('.')[0]
```

Reply:
Thank you. I was a little busy and postponed this MR.